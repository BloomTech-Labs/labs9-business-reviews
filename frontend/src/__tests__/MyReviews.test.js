import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import MyReviewsComponent from '../components/MyReviews';

describe('MyReviewsComponent />', () => {
  it('renders', () => {
    shallow(<MyReviewsComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<MyReviewsComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
