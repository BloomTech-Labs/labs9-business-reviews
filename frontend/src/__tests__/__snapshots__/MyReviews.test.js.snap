// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyReviewsComponent /> matches snapshot 1`] = `
<styled.div>
  <h1 />
  <styled.div />
</styled.div>
`;
